source/simulator/stokes_matrix_free.cc

```diff
@@ -660,6 +660,34 @@ namespace aspect
     }
   }
 
+
+  namespace MatrixFreeStokesOperators
+  {
+    template <int dim, typename number>
+    inline std::size_t
+    OperatorCellData<dim,number>::memory_consumption() const
+    {
+      return viscosity.memory_consumption()
+             + viscosity_derivative_wrt_pressure_table.memory_consumption()
+             + strain_rate_table.memory_consumption()
+             + viscosity_derivative_wrt_strain_rate_table.memory_consumption();
+
+    }
+
+    template <int dim, typename number>
+    void
+    OperatorCellData<dim,number>::clear()
+    {
+      enable_newton_derivatives = false;
+      // TODO: use Table::clear() once implemented in 10.0.pre
```

Comment:
Just got approved, should be able to do it soon!

Reply:
I leave this for a later time, if you don't mind.

Reply:
I mean after this PR got merged.